schemas/cloudToDevice/pgnss/pgnss.json

```diff
@@ -0,0 +1,19 @@
+{
+   "title": "PGNSS Success Response",
+   "description": "Responds with link to PGNSS file on success.",
+   "type": "array",
+   "prefixItems": [
+      {
+         "type": "string",
+         "description": "Host. The protocol agnostic URL for accessing PGNSS prediction files.",
+         "pattern": "pgps\\.nrfcloud\\.com"
```

Comment:
Add CNAME?

Reply:
Yes. This is done in [this PR](https://github.com/nRFCloud/backend/pull/2249).